### DISCUSSION TITLE
Filters import question

### DISCUSSION THREAD

**Discussion by Fehertukor** [Docile-Alligator/Infinity-For-Reddit#1327] [Help needed]
Hello,

I need a little bit of help with my filters.
I am planning to move from boost to infinity.
And my problem is the 10k+ filter list for subreddits and users too.
How would you import them without breaking the app?
For example can I import all my filtered subreddits into one rule ? (Or that will kill the app?)
Or should I separate in 1k groups?
What is your opinion?

Thanks!

**Comment by Wladefant** [comment]
Hey, here #1059 is a little discussion about filters. @user-727 seems to have looked at filter more carefully than I have ;). 
A [reddit discussion](https://www.reddit.com/r/Infinity_For_Reddit/comments/zvxvgt/cannot_seem_to_get_filters_working/) and [here](https://www.reddit.com/r/Infinity_For_Reddit/comments/xqwjg7/how_to_filter_out_subreddits/)

and [here](https://www.reddit.com/r/Infinity_For_Reddit/comments/x3l96c/the_limit_of_post_filters_and_why_being_to_block/) is something similar to your question.

**Comment by user-727** [comment]
Thanks for the ping @Wladefant

The issues I was experiencing post filters limit have been fixed with https://github.com/Docile-Alligator/Infinity-For-Reddit/commit/681a1c584c9e156d4f6fab9e19567a0b59e48736. Before I couldn't go over ≅1,500 users in a filter, now I'm at well over 2,500 and everything is working fine still, but I haven't tested with 10,000 yet.

You could copy your rules straight into the post filter UI in Infinity (just make sure your keyboard's clipboard isn't truncating anything), or you could export your settings (which contain your filters), edit them into a text editor and re-import them pretty easily too.

I don't think there's a need to seperate them into smaller groups anymore, but let me know how it goes!
